test/dom.html

```diff
@@ -41,158 +42,146 @@
 			</template>
 		</test-fixture>
 
-		<script>
-		describe('d2l-dom', function() {
-
-			var simpleFixture, wcFixture, mixedFixture;
-			var shadow;
-
-			beforeEach(function() {
-				simpleFixture = fixture('simpleFixture');
-				wcFixture = fixture('wcFixture');
-				mixedFixture = fixture('mixedFixture');
-				shadow = Polymer.Settings.dom === 'shadow' || Polymer.Settings.useShadow;
```

Comment:
ShadowDOM can still be disabled? Why remove this check?

Reply:
Good question.  Previously (with Polymer 1) it was necessary to perform certain tests different depending on whether shadow-dom was enabled or not.  With changes leading up to P3 (I then P1 -> P2), some of the API was made more consistent for shady & shadow.  In short, I think for certain tests it's no longer necessary (or desirable) to perform the text differently.  In these cases, I was able to perform the test the same regardless of shadow/shady.